src/ui/components/JEXLDebuggerPage.tsx

```diff
@@ -57,6 +76,47 @@ const JEXLDebuggerPage: FC = () => {
     }
   }, [jexlExpression, clientContext, addToast]);
 
+  const handleContextChange = useCallback(
+    (key: string, value: string, isText: boolean) => {
+      if (isText) {
+        setClientContext((prevContext) => ({
+          ...prevContext,
+          [key]: value,
+        }));
+      } else {
+        setTempContext((prevContext) => ({
+          ...prevContext,
+          [key]: value,
+        }));
+        debouncedParseAndSetContext(key, value);
```

Comment:
This is defined below. Im surprised TSC isnt complaining, because this should be undefined.